File: codeup/100000575/D.cpp
#include <iostream>
using namespace std;

int main() {
  int n, num, even_cnt, odd_cnt;
  while (scanf("%d", &n) != EOF) {
    even_cnt = odd_cnt = 0;
    while (n--) {
      scanf("%d", &num);
      if (num % 2)
        odd_cnt++;
      else
        even_cnt++;
    }
    if (even_cnt > odd_cnt)
      printf("NO\n");
    else
      printf("YES\n");
  }
  return 0;
}

File: codeup/100000575/D.py
#!/usr/bin/python3
import sys
while True:
    line = sys.stdin.readline()
    if not line:
        break
    n = int(line)
    even_cnt = odd_cnt = 0
    for num in list(map(int, input().split())):
        if num % 2:
            odd_cnt += 1
        else:
            even_cnt += 1

    if even_cnt > odd_cnt:
        print("NO")
    else:
        print("YES")
